### PR TITLE
Merge Level select level description into Main

### PR DESCRIPTION
**Pull request by mthe371**
This pull request adds the level description template for the level description and module.css files for the nav bar and level description
Partially resolves Level Select - Level Description Layout - FE  #7 

TODO: Currently placeholder info showing so requires linking and insert the component into the level description/discussion tab

### CODE REVIEW

File: src/components/NavigationBar/NavigationBar.module.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
+.root {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
```
Comment (mlu415) [inline src/components/NavigationBar/NavigationBar.module.css:4]: can u remove this? seems to be causing the components to get pushed down

Comment (AoMedium) [reply src/components/NavigationBar/NavigationBar.module.css:4]: seems to be resolved in #50

**Review by AoMedium — APPROVED** [review]
Looks good with #50 changes, will merge now and followup with a fix PR (see Discord discussion)

**Review by mlu415 — APPROVED** [review]
ok lets go

**Pull request merged**